### PR TITLE
docs: read row header from repoStatus CSV file

### PR DESCRIPTION
**Pull request by eyevana**
## What I did

1. Added one more column to the `repoStatus.csv` to serve as a row ID. The second column in the CSV file will now act as the row's header. In some [cases](https://xd.adobe.com/view/6f3078cf-c5d5-4c7b-8e30-509acd11525c-3a63/), it's the title/alias of the element, but in [others](https://xd.adobe.com/view/7728dcb1-2285-45ee-92ee-41998615aa57-227b/) it's used to indicate one of its many flavors
2. Updated the CSV file according to the XD files I found in our [tracking doc](https://docs.google.com/document/d/1P0bTHjrNw6dkFkC8Kq8HBwZKjKOD9Jkg2dasK9FPbv4/edit#).

## Testing Instructions

1. Cross check the **Overview/Repo status** section for the following elements against the [DP](https://deploy-preview-1057--red-hat-design-system.netlify.app/):

- [Statistic](https://xd.adobe.com/view/6f595e3f-c8ff-466b-8042-4c3a8a5776fa-3a0a/)
- [Spinner](https://xd.adobe.com/view/efcbdd74-c608-4a42-ad07-b009d45534d8-ba98/)
- [Footer](https://xd.adobe.com/view/7dceb708-6c56-49f0-a6bc-a01d388ba872-289d/)
- [Blockquote](https://xd.adobe.com/view/05880b69-8093-47c6-91c0-fded6b35fa73-50db/)
- [Badge](https://xd.adobe.com/view/f3089ecd-1286-4622-8a8f-548e9462d730-a65b/)
- [Pagination](https://xd.adobe.com/view/ad3bcf77-23b2-45e4-997d-f711a87173b1-f868/)
- [Secondary nav](https://xd.adobe.com/view/75edef5f-3cfa-4cc3-b12e-a1e3d5d37b5b-20be/)
- [Tooltip](https://xd.adobe.com/view/df265b71-7bc3-47b6-bd8a-450f1d44212a-8a75/)
- [Dialog](https://xd.adobe.com/view/615c97ab-d017-4473-9daa-8aa00b1e3348-340f/)
- [Avatar](https://xd.adobe.com/view/6f3078cf-c5d5-4c7b-8e30-509acd11525c-3a63/)
- [Tag](https://xd.adobe.com/view/11e76e09-10f4-44af-b74b-1f3e562970e5-44b4/)
- [Accordion](https://xd.adobe.com/view/ba7a7209-4d1f-449d-a3f9-042a0c7ebda8-ef29/)
- [Tabs](https://xd.adobe.com/view/d42498d5-d4fa-4c23-a8b5-99ccba266bfe-8177/)
- [Button](https://xd.adobe.com/view/7728dcb1-2285-45ee-92ee-41998615aa57-227b/)
- [Call to Action](https://xd.adobe.com/view/fe8152fb-ec3c-47f6-bcba-55a2a3ce61c5-abd2/)

## Notes to Reviewers

1. @coreyvickery I wasn't able to get a hold of the XD for `Alert` to validate my changes. Can you share that with me, please? Also, can you help me with the other rows in the CSV files that don't correspond with a published element? Not sure where those show up or if they're old entries we need to remove.

### CODE REVIEW

File: docs/_data/repoStatus.csv [hunk 1]
```diff
@@ -1,74 +1,85 @@
...
-Card - name slider,,,x,,
-Avatar,x,,,,
-Tag - Red - filled and unfilled,x,,,,
-Tag - Orange - filled and unfilled,x,,,,
-Tag - Green - filled and unfilled,x,,,,
-Tag - Cyan - filled and unfilled,x,,,,
-Tag - Blue - filled and unfilled,x,,,,
-Tag - Purple - filled and unfilled,x,,,,
-Tag - Gray - filled and unfilled,x,,,,
+id,Element/Pattern,RHDS,WebRH,WebDMS,Adobe Target,
```
Comment (zeroedin) [inline docs/_data/repoStatus.csv:1]: I would alpha order this file by the id field now.  That way rows are easier to find and update

**Review by zeroedin — APPROVED** [review]

File: docs/_data/repoStatus.csv (hunk 1 above)
Comment (eyevana) [reply docs/_data/repoStatus.csv:1]: Great idea! Thanks ✅

File: docs/_data/repoStatus.csv
Comment (bennypowers) [inline docs/_data/repoStatus.csv:1]: Holding this comment for @coreyvickery's review: please press "Resolve conversation" to merge the changes

File: docs/_data/repoStatus.csv [hunk 2]
```diff
@@ -1,74 +1,85 @@
...
+Tabs,Boxed,x,,,,
+Tag,Red - filled and unfilled,x,,,,
+Tag,Orange - filled and unfilled,x,,,,
+Tag,Green - filled and unfilled,x,,,,
+Tag,Cyan - filled and unfilled,x,,,,
+Tag,Blue - filled and unfilled,x,,,,
+Tag,Purple - filled and unfilled,x,,,,
+Tag,Gray - filled and unfilled,x,,,,
+Tooltip,Tooltip,x,,,,
+,12-column grid,,x,x,,
```
Comment (eyevana) [inline docs/_data/repoStatus.csv:62]: @coreyvickery I believe lines 62-85 is data for the repo status tables that we should be displaying under Patterns. Do you want to break this out into a separate issue for us to clean up?

Comment (coreyvickery) [reply docs/_data/repoStatus.csv:62]: @eyevana Yes please.

Comment (eyevana) [reply docs/_data/repoStatus.csv:62]: Thank you!

**Review by coreyvickery — APPROVED** [review]

**Pull request merged**